WTWSMS/history/characters/roman.txt

```diff
@@ -1985,17 +1985,13 @@
 	420.1.1 = {
 		birth="420.1.1"
 	}
-	492.1.1 = {
-	name="Gelasius"
-	# AKA: Acacius
-	}
 	496.11.21 = {
 		death="496.11.21"
 	}
 }
 
 7769 = {
-	name="Acacius"
+	name="Felix Anicius"
```

Comment:
❓

Reply:
Similar to above.

Reply:
Same as above.

Reply:
Is Anicius in reference to the dynasty? Then that should not be included like that.

Reply:
It was part of his name, but now I think know why it may look bad in game (Felix Anicius Anicius) :P I will remove the "Anicius" part